### PR TITLE
ignore the CVE report for rancher/library-traefik:2.4.8

### PR DESCRIPTION
**Pull request by jiaqiluo**
ignore the CVE report for rancher/library-traefik:2.4.8 as the upstream will NOT update the package version as traefik doesn't use the specific part that has a vulnerability: https://github.com/traefik/traefik/issues/8283

### CODE REVIEW

**Comment by deniseschannon** [comment]
Does this mean that it's included in the report with justification of why we aren't updating it?

**Comment by jiaqiluo** [comment]
@deniseschannon  The purpose of this PR is to prevent the bot from making/opening issues for this particular CVE in this particular image. 

We cannot update the image because it is a mirror of the upstream image and the upstream will not fix it. comment in the upstream: https://github.com/traefik/traefik/issues/8283#issuecomment-883790303

**Comment by deniseschannon** [comment]
How are we planing on tracking this justification in our report if we start ignoring these images? What happens when other vulnerabilities come through for that specific image?  @rmweir @cbron

@jiaqiluo This isn't meant to pick on this particular image, but just seeing potential holes in the process.

**Comment by jiaqiluo** [comment]
Your questions make sense to me. 
I'd like to wait for @rmweir or @cbron to provide more insights.

**Comment by cbron** [comment]
> How are we planing on tracking this justification in our report if we start ignoring these images? 

@deniseschannon  We aren't ignoring the image, we are ignoring a CVE within the image. I believe we do have a way to ignore images, but this isn't using that solution.

**Comment by deniseschannon** [comment]
Yep, Jack's comment help clarified that we are ignoring that specific CVE for the scanning, but will that automatically be in the report or how do we make his note into the report itself as justification? @cbron

File: docs/_data/cves-v2.6.csv [hunk 1]
```diff
@@ -262,7 +262,7 @@ rancher/klipper-helm:v0.6.1-build20210616,github.com/opencontainers/runc,alpine,
 rancher/klipper-helm:v0.6.1-build20210616,github.com/opencontainers/runc,alpine,CVE-2019-19921,HIGH,https://avd.aquasec.com/nvd/cve-2019-19921,v1.0.0-rc9.0.20200122160610-2fc03cc11c77,,waiting,Upstream issue: https://github.com/helm/helm/issues/9956
 rancher/klipper-lb:v0.1.2,musl,alpine,CVE-2019-14697,CRITICAL,https://avd.aquasec.com/nvd/cve-2019-14697,1.1.19-r11,,triaged,
 rancher/klipper-lb:v0.1.2,musl-utils,alpine,CVE-2019-14697,CRITICAL,https://avd.aquasec.com/nvd/cve-2019-14697,1.1.19-r11,,triaged,
-rancher/library-traefik:2.4.8,github.com/gogo/protobuf,alpine,CVE-2021-3121,HIGH,https://avd.aquasec.com/nvd/cve-2021-3121,v1.3.2,,triaged,
+rancher/library-traefik:2.4.8,github.com/gogo/protobuf,alpine,CVE-2021-3121,HIGH,https://avd.aquasec.com/nvd/cve-2021-3121,v1.3.2,,ignore,The upstream will NOT update the package version as traefik doesn't use the specific part that has a vulnerability: https://github.com/traefik/traefik/issues/8283
```
Comment (cbron) [inline docs/_data/cves-v2.6.csv:265]:
```suggestion
rancher/library-traefik:2.4.8,github.com/gogo/protobuf,alpine,CVE-2021-3121,HIGH,https://avd.aquasec.com/nvd/cve-2021-3121,v1.3.2,,ignore, traefik does not use the package that includes the vulnerability and therefore will not be updated: https://github.com/traefik/traefik/issues/8283

```

**Comment by cbron** [comment]
Whatever we put in this PR is what will end up in the report. The report is just the csv, which is also what the scanner runs off of.

File: docs/_data/cves-v2.6.csv (hunk 1 above)
Comment (jiaqiluo) [reply docs/_data/cves-v2.6.csv]: fixed.

**Review by cbron — APPROVED** [review]

**Pull request merged**